### PR TITLE
Make Wishiwashi (School) an evolution

### PR DESCRIPTION
**Pull request by Ultima1990**
Wishiwashi's ability, Schooling, transforms Wishiwashi (Solo) into Wishiwashi (School) if it's at or above level 20 and its HP is at or above 25%. In clicker this can be represented as a level 20 evolution.

Yes. This makes Wishiwashi (School) easier to get. But also this actually nerfs Wishiwashi (School). By making it an evolution the School form also gets increased egg steps, which it kind of needs. With its absurd stats and Solo's very low steps Wishiwashi (School) was looking like it was going to be an insane powerhouse, especially with max protein. Not Gyarados level, but close.

Also increased Totem Wishiwashi (School)'s steps to match the non-totem.

Gyarados reaches efficiency 9 in Alola btw. Thought someone'd be interested by that.

### CODE REVIEW

File: src/scripts/pokemons/PokemonList.ts [hunk 1]
```diff
@@ -16379,7 +16380,7 @@ const pokemonList = createPokemonArray(
         'id': 746.02,
         'name': 'Totem Wishiwashi (School)',
         'type': [PokemonType.Water],
-        'eggCycles': 15,
+        'eggCycles': 23,
```
Comment (RedSparr0w) [inline src/scripts/pokemons/PokemonList.ts:16383]: Does this actually affect it's egg steps?

Comment (Ultima1990) [reply src/scripts/pokemons/PokemonList.ts:16383]: Yes it's 920 up from 600.
WishiwashiSteps

Comment (RedSparr0w) [reply src/scripts/pokemons/PokemonList.ts:16383]: Ah sorry, didn't realise this was for the totem variant 👍

**Review by RedSparr0w — APPROVED** [review]
Thanks for the PR! 👍

**Pull request merged**